docs/admin-guide/audit/audit.md

```diff
@@ -34,12 +34,18 @@ The [official documentation describes](https://kubernetes.io/docs/tasks/debug-ap
 | `audit-log-maxsize`  | Specifies the maximum size in megabytes of the audit <br/>log file before it gets rotated <br/>Sets the `--audit-log-maxsize` flag.                   |
 | `audit-log-format`   | Specifies the logging format used.<br/>Options are:<br/>`"legacy"` indicates 1-line text format for each event <br/> `"json"` indicates a structured json format. <br/>Sets the `--audit-log-format` flag.                   |
 
-These are injected into the cluster at creation time via `kubeadmConfigPatches` that are supplied to `kubeadm`.
+These are injected into the cluster at creation time.
```

Comment:
I would move these entirely to source code as it is more applicable for developers, then it is for administrators who are consuming and configuring it.

Reply:
Agreed. Removed in 711b01c0d1cf5f97ab0c0b9051264b8a2b006edd